justfile

```diff
@@ -101,10 +104,11 @@ generate cli=default_cli platform=default_platform:
     nix run -L .#{{ cli }} -- generate \
         --workspace-dir ./{{ workspace_dir }} \
         --image-replacements ./{{ workspace_dir }}/just.containerlookup \
-        --reference-values {{ platform }}\
+        --reference-values {{ platform }} \
+        --genpolicy-cache-path ./{{ workspace_dir }}/../layers-cache.json \
```

Comment:
Ah sorry, I oversaw this. But generally we currenly save the layers-cache in the workspace folder, which we clean up on every `just default` command and I wanted some deployment speed. Since this is the justfile we might discuss doing such a change (in another PR) though.